pwa/src/components/handlers/handlerForm.tsx

```diff
@@ -138,8 +138,7 @@ export default function HandlerForm({id, endpointId}) {
       template: event.target.template.value
         ? event.target.template.value : null,
       templateType: event.target.templateType.value
-        ? event.target.templateType.value : null,
-      conditions,
+        ? event.target.entity.value : null,
```

Comment:
You're doing a couple of things in this line:

1. You're setting the `templateType` key within the `body` object (seems good);
2. You're checking if `event.target.templateType.value` (seems good);
3. If the value exists you're setting `event.target.ENTITY.value` (**does not seem right)** and finally
4. If the value did not exist you're setting it to `null` (seems good).

Currently, you're filling the key with irrelevant data, and because of that the it doesn't behave as intended.